package.json

```diff
@@ -3,16 +3,27 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@date-io/date-fns": "^2.14.0",
+    "@date-io/dayjs": "^2.14.0",
+    "@date-io/luxon": "^2.14.0",
+    "@date-io/moment": "^2.14.0",
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
     "@mui/icons-material": "^5.8.4",
-    "@mui/material": "^5.8.4",
+    "@mui/material": "^5.8.6",
+    "@mui/styled-engine-sc": "^5.8.0",
     "@mui/styles": "^5.8.4",
+    "@mui/x-data-grid": "^5.12.3",
+    "@mui/x-data-grid-generator": "^5.12.3",
+    "@mui/x-data-grid-premium": "^5.12.3",
+    "@mui/x-date-pickers": "^5.0.0-alpha.7",
     "axios": "^0.27.2",
+    "date-fns": "^2.28.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.5",
```

Comment:
Any reason to include this library as we are using MUI styled library for css/enhancements?

Reply:
I did not do this intentionally and also don't have idea.

Reply:
This library is another way to style the components but we are using MUI styled which does the same thing. You can remove this if not using.

Reply:
removed